DependencyInjection/Providers/GoogleProviderConfigurator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace KnpU\OAuth2ClientBundle\DependencyInjection\Providers;
+
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class GoogleProviderConfigurator implements ProviderConfiguratorInterface
+{
+    public function buildConfiguration(NodeBuilder $node)
+    {
+        // no custom options
```

Comment:
@weaverryan Right hostedDomain and accsesType just optional parameters. But we must add this your right my bad :+1: